### PR TITLE
Added team bios and community member links to /about page

### PR DESCRIPTION
**Pull request by jakerockland**
- Fixes: https://github.com/gitcoinco/web/issues/222

##### Description

Adds team and community profiles to the /about page, as per #222 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

n/a

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/222

### CODE REVIEW

File: app/retail/templates/about.html [hunk 1]
```diff
@@ -16,6 +16,39 @@
 {% endcomment %}
 {% load i18n static %}
 <!DOCTYPE html>
+<style type="text/css">
```
Comment (thelostone-mc) [inline app/retail/templates/about.html]: @jakerockland Could we move this into it's own css file ? 
Bonus: You could port over the CSS needed only for `about.html` from `base.css` / `gitcoin.css` to the newly created `about.css`

That way linting picks up any errors in the css

File: app/retail/templates/about.html [hunk 2]
```diff
@@ -16,6 +16,39 @@
...
+  div.teamProfileLeft, div.teamProfileRight {
```
Comment (thelostone-mc) [inline app/retail/templates/about.html]: Could you break this into two lines ? (the style-lint will point it out :P )

File: app/retail/templates/about.html [hunk 3]
```diff
@@ -16,6 +16,39 @@
...
+    background-color: #eff4ee;
+    padding: 1em;
+    margin-bottom: 2em;
+    box-shadow: 0px 2px 5px 0px grey;
```
Comment (thelostone-mc) [inline app/retail/templates/about.html]: Could we use the hexcode instead to ensure the same color is rendered across all browsers ?

File: app/retail/templates/about.html [hunk 4]
```diff
@@ -99,60 +132,91 @@ <h6 style="text-align: center;" >{% trans "Who, what, when, where, and most impo
...
-        <div class="col p-5 animated fadeInRightShort ">
-          <div class="row">
-            <h4 class="pt-1">{% trans "Post Launch (2017+)" %}</h4>
+        <div class="col pb-4 animated text-center fadeInRightShort">
+          <div class="teamProfileRight">
+            <img class="teamProfileAvatar img-fluid" src="{% static "v2/images/team/mark-beacom.jpg" %}">
+            <p><span class="font-bigger-1 font-weight-300"><strong>Mark Beacom</strong></span><br><span class="font-smaller-1"><em>Engineering</em></span></p>
+            <a href="https://gitcoin.co/profile/mbeacom"><img class="teamProfileSocialImg" src="{% static "v2/images/social/gitcoin.png" %}"></a>
+            <a href="https://github.com/mbeacom"><img class="teamProfileSocialImg" src="{% static "v2/images/social/github.png" %}"></a>
+            <p><strong>Favorite Gitcoin Feature:</strong> Start/Stop Work<br><strong>Favorite Food:</strong> Dolsot Bibimbap</p>
```
Comment (thelostone-mc) [inline app/retail/templates/about.html]: @mbeacom Thoughts on breaking this into multiple lines ? (for readability )

Comment (mbeacom) [reply app/retail/templates/about.html]: @thelostone-mc I agree 👍

**Comment by thelostone-mc** [comment]
@jakerockland If you could throw in a screenshot, that would be super helpful ^_^

**Comment by owocki** [comment]
i see a bunch of broken images on this when i pull it down.  did you forget to add team photos?  static/v2/images/social/gitcoin.png is missing too

**Comment by jakerockland** [comment]
@owocki Ah I didn't realize I didn't check in the added images. I have checked them in but as a side note it looks like `app/static/*` is in the `.gitignore` file, is that right?

**Comment by owocki** [comment]
@jakerockland they should be in `app/assets/*` not `app/static/*`

**Comment by owocki** [comment]
mind putting in a fix for [this responsive design issue](http://bits.owocki.com/2B0w1b1g1D0R/Screen%20Recording%202018-04-16%20at%2009.20%20AM.gif) ?

**Comment by thelostone-mc** [comment]
Could we define these as array of objects in python as just run it as a loop via djangos templating engine (like how we do it for  testimonial / issues on dashboard and so on) as against to hardcoding it into html ? 

@mbeacom your thoughts?

File: app/static/v2/css/about.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
+/***********************
+* Style for about.html *
+***********************/
```
Comment (thelostone-mc) [inline app/static/v2/css/about.css]: these aren't needed ^_^
The filename is good enough !!

**Comment by owocki** [comment]
> Could we define these as array of objects in python as just run it as a loop via djangos templating engine (like how we do it for testimonial / issues on dashboard and so on) as against to hardcoding it into html ?

this is probably a good idea.. this way we can drive the information via the database down the line (especially important for the community section).  

jake, happy to throw in a little tip on top of this since i think this has been more code review than you were asking for :P

**Comment by thelostone-mc** [comment]
Haha sorry about the code review !! 

If you need help figuring it out, reach out to us ^_^

**Comment by jakerockland** [comment]
@owocki I fixed the directory issue (I hadn't realized that `app/assets/*` --> `app/static/*`, thank you!) and I _believe_ I fixed the responsive issue you were pointing out.

There is a more pervasive responsive strangeness that wasn't introduced in this PR, which I have attached a screenshot of here of there being more spacing on the right than on the left when in mobile. I think this is out of the scope of this PR though. Hopefully I fixed the bug you were pointing out though (the blocks becoming staggered in a weird way), otherwise I at least fixed a bug that needed to be fixed 😅.

screenshot 2018-04-16 15 12 39

I can look into the Django rendering as well! Need to poke around templates on dashboard a bit more to better understand this. I'll never say no to a tip hehe.

**Comment by jakerockland** [comment]
@vs77bb what is your favorite food plz? 😄

**Comment by owocki** [comment]
> I have made the temporary executive decision that it is "Tangerine Gelato"

lol

> I can look into the Django rendering as well! Need to poke around templates on dashboard a bit more to better understand this. I'll never say no to a tip hehe.


[the tools view](https://github.com/gitcoinco/web/blob/master/app/dashboard/views.py#L819-L1125) is a good example of this (albeit a long one). basically just define a dict/array structure in the view and pass it down into the `TemplateResponse` with `context`

**Comment by jakerockland** [comment]
@owocki @thelostone-mc Templating done as requested. 😄

Note I've created separate dictionaries for the left/right columns as it seems to be much more readable code to me and also implicitly conveys that the number of community/team members should be an even number to make the columns equal.

I believe all issues raised on this PR have been addressed and we're good for merge (🙏). Thoughts?

**Comment by owocki** [comment]
just put in some fixes.. afcec453bdc8b3b82959962c34fcba2b8c34e48b 

LGTM

**Comment by jakerockland** [comment]
Haha I really like this stickler-ci, haven't seen that before but its awesome. 👍

**Comment by mbeacom** [comment]
This PR still had a lingering `static/` file 😅 https://github.com/gitcoinco/web/tree/master/app/static/v2/images/social

**Comment by jakerockland** [comment]
Ugh, I thought I removed that. Sorry about that 😓

**Comment by mbeacom** [comment]
@jakerockland All good! 😄

**Pull request merged**